### ISSUE TITLE
Moon looks too bright

### ISSUE THREAD

**Issue by EricNelson28** [Stellarium/stellarium#3176]
The Moon looks too bright and washed out especially in daylight setting on the newest version.
The Moon never looks that bright during the daytime no matter what phase.
Previous versions accurately captured how the Moon looks during the day.
The new version does a good job with eclipse shadows though.
![stellarium-002](https://user-images.githubusercontent.com/130680523/231802071-b29e957c-cb0b-4fcd-819c-677edf0a3f3a.png)

**Comment by gzotti** [comment]
Yes, I feel we are refining that every year. Some change improves something but breaks brightness, another change fixes...

**Comment by alex-w** [comment]
Duplicate of #3067

**Comment by EricNelson28** [comment]
Yes, just found out.

Now while the eclipse shadows were improved, the lighting adaptation of them when it comes to lunar eclipses isn't.
The Moon's brightness should affect how Earth's umbral shadow looks depending on the umbral magnitude of the eclipse.
Many people don't see redness much until a lot of the Moon's in the shadow.

That should be another example of dynamic eye adaptation, as well as the Moon looking very dark during a total solar eclipse.

During a total solar eclipse, the Moon looks almost like a black hole up close, and the background sky should look very dark especially when looking at the Moon up close as the solar corona still looks bright.
And the corona image should also be affected by Earth's atmosphere (more flattened nearer to the horizon).

Always working on things for the better.
